frontend/src/components/WindSettings.jsx

```diff
@@ -413,10 +183,8 @@ WindSettings.propTypes = {
     selectedWindType: PropTypes.string.isRequired,
     fluctuationPercentage: PropTypes.number.isRequired,
     windShears: PropTypes.array.isRequired,
-    addNewWindShear: PropTypes.func.isRequired,
-    windShearData: PropTypes.object.isRequired,
```

Comment:
@syedomair0 Also remove anything related to Wind Shear from all files, because its all going to be dead code in new version